api/views.py

```diff
@@ -244,13 +244,20 @@ class MutationRetrieve(generics.RetrieveAPIView):
 
 # Samples
 
+# via https://github.com/carltongibson/django-filter/issues/137#issuecomment-38158832
+class ListFilter(django_filters.Filter):
+    def filter(self, qs, value):
+        value_list = value.split(u',')
+        return super(ListFilter, self).filter(qs, django_filters.fields.Lookup(value_list, 'in'))
+
 class SampleFilter(filters.FilterSet):
     age_diagnosed__gte = django_filters.IsoDateTimeFilter(name='age_diagnosed', lookup_expr='gte')
     age_diagnosed__lte = django_filters.IsoDateTimeFilter(name='age_diagnosed', lookup_expr='lte')
+    all_mutations = ListFilter(name='mutations__gene')
```

Comment:
This would make the query parameter mutations__gene, are you filtering on the associated gene object(s)?